inference-engine/tests/functional/plugin/cpu/single_layer_tests/slice.cpp

```diff
@@ -0,0 +1,549 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <shared_test_classes/single_layer/strided_slice.hpp>
+#include "ngraph_functions/builders.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+
+using namespace CPUTestUtils;
+using namespace LayerTestsDefinitions;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+
+typedef std::tuple<
+    std::vector<InputShape>,         // Parameters shapes
+    Slice8SpecificParams,            // Slice-8 specific parameters
+    ElementType,                     // Network precision
+    CPUSpecificParams                // CPU specific parameters
+> Slice8LayerTestCPUParam;
+
+class Slice8LayerCPUTest : public testing::WithParamInterface<Slice8LayerTestCPUParam>,
+                                 virtual public SubgraphBaseTest, public CPUTestsBase {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<Slice8LayerTestCPUParam> obj) {
+        std::vector<InputShape> shapes;
+        Slice8SpecificParams params;
+        ElementType netPrecision;
+        CPUSpecificParams cpuParams;
+        std::tie(shapes, params, netPrecision, cpuParams) = obj.param;
+
+        std::ostringstream result;
+        result << "IS=(";
+        for (const auto& shape : shapes) {
+            result << CommonTestUtils::partialShape2str({shape.first}) << "_";
+        }
+        result << ")_TS=(";
```

Comment:
I think we should print target shapes after corresponding dynamic shape. Otherwise it is not clear to which dynamic shape the target shape corresponds

Reply:
In general, it make sense to divide shapes by execution groups, but here just one dynamic input thus this implementation is enough.